package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/Brexbot/DiscordBot#readme",
   "type": "commonjs",
   "devDependencies": {
+    "@types/cron": "^2.0.0",
```

Comment:
I think it might be good to add the actual cron library to the dependencies.

Reply:
Good catch, this is why you don't code first thing in the morning with no coffee.